frame/balances/src/lib.rs

```diff
@@ -220,11 +220,14 @@ pub mod pallet {
 
 		pub struct TestDefaultConfig;
 
-		#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
+		#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig, false)]
```

Comment:
This looks like a pretty terrible syntax. What does `false` even mean here? Do I have to go and read the documentation for every change I want to make?

Reply:
I suggest switching over to a keyword that makes sense, and instead of explicitly specifying true or false, have the existence of the attribute be the true state, while an absence of it represents false.

Reply:
Makes sense. What do you think about `no_runtime_injection`? Any other suggestions?

Reply:
What does `injection` mean in this context? I think something like `no_aggregated_types` would definitely signal the intent better.

Reply:
yeah I would agree that the whole style of like `some_func(true, false, 33, "hey");` has always been a smell.  At least with proc macros we have the opportunity to make these more understandable 👍🏻